testing/cucumber/features/linters/background_does_more_than_setup.feature

```diff
@@ -48,3 +48,35 @@ Feature: Background does more than setup linter
       """
     When it is linted
     Then no error is reported
+
+  Scenario Outline: Configuration of keywords for different dialect
+    Given a linter for backgrounds that do more than setup has been registered
+    And the following configuration file:
+      """
+      BackgroundDoesMoreThanSetupLinter:
+        Given:
+          - Dado
+        When:
+          - Quando
+        Then:
+          - Então
+      """
+    And the following feature:
+      """
+      # language:pt
+      Funcionalidade: Feature name
+
+        Contexto:
+          Dado some setup in pt dialect
+          <step>
+      """
+    When the configuration file is loaded
+    And it is linted
+    Then an error is reported:
+      | linter                            | problem                        | location         |
+      | BackgroundDoesMoreThanSetupLinter | Background has non-setup steps | <path_to_file>:4 |
+
+    Examples:
+      | step                                      |
+      | Quando this is an action in pt dialect    |
+      | Então this is an validation in pt dialect |
```

Comment:
Could we get a third example row that uses another keyword and add that to either the `When` or `Then` in the config file? That way it is more obvious to users that the configuration is an array that can have more than one word.